builder/internals.go

```diff
@@ -433,7 +434,7 @@ func (b *Builder) pullImage(name string) (*imagepkg.Image, error) {
 	job.SetenvBool("json", b.StreamFormatter.Json())
 	job.SetenvBool("parallel", true)
 	job.SetenvJson("authConfig", pullRegistryAuth)
-	job.Stdout.Add(b.OutOld)
+	job.Stdout.Add(ioutils.NopWriteCloser(b.OutOld))
```

Comment:
Oh wait - maybe you're confused by the name `NopWriteCloser`. This makes only the `Close` call be a nop - the writes still work and are written to the underlying writer.